resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -37,15 +37,9 @@ fi
 
 rm -f "$PIDFILE"
 
-if [ ! -e "$restartfile" ]; then
-    mkdir -p "$dir"
-    echo -n "0" > "$restartfile"
-elif [ ! -r "$restartfile" ] || [ ! -w "$restartfile" ]; then
-    echo "The restart-file at <%= EZBake::Config[:restart_file] %> is not readable and/or writeable." 1>&2
-    exit 1
-fi
+init_restart_file "$restartfile" || exit $?
 
-${JAVA_BIN} ${JAVA_ARGS} -Djava.security.egd=/dev/urandom -cp ${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %> clojure.main -m <%= EZBake::Config[:main_namespace] %> --config "${CONFIG}" -b "${BOOTSTRAP_CONFIG}" &
+${JAVA_BIN} ${JAVA_ARGS} -Djava.security.egd=/dev/urandom -cp ${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %> clojure.main -m <%= EZBake::Config[:main_namespace] %> --config "${CONFIG}" -b "${BOOTSTRAP_CONFIG}" -r "${restartfile}" &
```

Comment:
I prefer long flags to short flags so it's obvious what the flag is doing, but 🍤 .